### PR TITLE
First draft of materials page

### PR DESCRIPTION
**Pull request by HerrBertling**
Moin :)

I did a quick first draft for the materials page as described in the [brainstorming document](https://github.com/CSSclasses/CSSclasses/wiki/November-2016:-Brainstorming#materials)

* Has a small menu generated from the different posts (which are the material subpages)
* Has previous/next buttons for easier navigation within the materials
* I threw out some JS code that had a scroll event listener. Wasn't needed.

There is probably still a lot of unused JS/CSS in there.

image

image

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+
+# Site settings
+title: CSSclasses Materials
+email: contact@cssconf.eu
```
Comment (Schubidu) [inline _config.yml:5]: change the mail to something with `@cssclass.es`

Comment (lukaszklis) [reply _config.yml:5]: For now let's leave it like this, we'll change it after we take care of the "org" stuff. :)

**Comment by lukaszklis** [comment]
First of all, big thanks for looking into this. :heart:

I'll assign it to myself to click it through later today.

**Comment by HerrBertling** [comment]
Cool, thanks @lukaszklis

As you might notice, this is basically a copy of the CSSClasses website with some adjustments and some deleted files ;)

**Comment by lukaszklis** [comment]
@HerrBertling sorry, didn't manage to click it through yesterday. :/ Let's merge it and create follow up issues whenever necessary.

**Comment by HerrBertling** [comment]
🎉

**Pull request merged**